src/pages/index.astro

```diff
@@ -96,6 +96,16 @@ import { Content as Sponsor } from "./_sponsor.md";
     line-height: 1;
     font-weight: 900;
 
+    em {
```

Comment:
em is not needed here as I do believe it's globally defined and font-weight for the heading is already 900 and the headings family is already neo grotesque.

Reply:
Ah, thank you for pointing that out. Learning.

I've pushed the change addressing this. Even though headings family is neo, it was only defined and not applied to all headings, hence kept the font-family thing as is.

<img width="296" alt="Screenshot 2024-03-28 at 10 34 16 PM" src="https://github.com/queen-raae/datainthewild.fm/assets/46300090/144245d5-9309-483a-9ef4-8542b911354f">